integrations/helm/release/release.go

```diff
@@ -293,3 +315,31 @@ func (r *Release) annotateResources(release *hapi_release.Release, fhr ifv1.Flux
 func fhrResourceID(fhr ifv1.FluxHelmRelease) flux.ResourceID {
 	return flux.MakeResourceID(fhr.Namespace, "FluxHelmRelease", fhr.Name)
 }
+
+
+// Merges source and destination map, preferring values from the source map
+func mergeValues(dest, src chartutil.Values) chartutil.Values {
```

Comment:
Please add a comment and link to the original code 
https://github.com/helm/helm/blob/master/cmd/helm/install.go#L330